tests/Handler/CreateAndFetchTest.php

```diff
@@ -74,7 +74,7 @@ public function handleShouldExecuteTheCommandAndReturnAnEmptyResponse(): void
             $this->idGenerator
         );
 
-        $request = new ServerRequest();
```

Comment:
Please add a new test scenario since we still have to cover the previous behaviour (where generated identifier is used).